CONTRIBUTING.md

```diff
@@ -91,21 +77,88 @@ To see test coverage, run `dotnet test` from a console window and you will see t
 
 Or, after running the tests, open the file `TestResults\Results\index.htm` in a browser.
 
-### Coding style
+## Pull Requests
+
+### How to Send Pull Requests
+
+Everyone is welcome to contribute code to `opentelemetry-dotnet` via GitHub
+pull requests (PRs).
+
+To create a new PR, fork the project in GitHub and clone the upstream repo:
+
+```sh
+$ git clone https://github.com/open-telemetry/opentelemetry-dotnet.git
+```
+
+Add your fork as an origin:
+
+```sh
+$ git remote add fork https://github.com/YOUR_GITHUB_USERNAME/opentelemetry-dotnet.git
+```
+
+Run tests:
+
+```sh
+dotnet test
+```
+
+Check out a new branch, make modifications and push the branch to your fork:
+
+```sh
+$ git checkout -b feature
+# edit files
+$ git commit
+$ git push fork feature
+```
+
+Open a pull request against the main `opentelemetry-dotnet` repo.
+
+### How to Receive Comments
+
+* If the PR is not ready for review, please put `[WIP]` in the title, tag it
+  as `work-in-progress`, or mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+* Make sure CLA is signed and CI is clear.
+
+### How to Get PRs Merged
+
+A PR is considered to be **ready to merge** when:
+* It has received approval from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver).
+  / [Maintainers](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer).
+* Major feedbacks are resolved.
+* It has been open for review for at least one working day. This gives people
+  reasonable time to review.
+* Trivial change (typo, cosmetic, doc, etc.) doesn't have to wait for one day.
+* Urgent fix can take exception as long as it has been actively communicated.
+
+Any Approver / Maintainer can merge the PR once it is **ready to merge**.
```

Comment:
only Maintainer can merge.

Reply:
Wonder if it is a good idea to align with other repo?
For example: https://github.com/open-telemetry/opentelemetry-go/blob/master/CONTRIBUTING.md#how-to-get-prs-merged

Reply:
There was a discussion about Who can merge PRs.. (trying to find link), and IIRC, it was decided only maintainers can do the actual "merge" to control release cadence. etc.

Reply:
https://github.com/open-telemetry/community/pull/308 Lets follow this guideline now

Reply:
Updated.